intake_xarray/__init__.py

```diff
@@ -41,3 +41,32 @@ def open(self, urlpath, chunks, **kwargs):
             chunks=chunks,
             xarray_kwargs=source_kwargs,
             metadata=base_kwargs['metadata'])
+
+
+class ZarrPlugin(base.Plugin):
+    """zarr>xarray reader"""
+
+    def __init__(self):
+        super(ZarrPlugin, self).__init__(
+            name='zarr',
+            version=__version__,
+            container='xarray',
+            partition_access=True
+        )
+
+    def open(self, urlpath, storage_options=None, **kwargs):
+        """
+        Parameters
+        ----------
+        urlpath: str
+            Path to source. This can be a local directory or a remote data
+            service (i.e., with a protocol specifier like ``'s3://``).
+        storage_options: dict
+            Parameters passed to the backend file-system
+        kwargs:
+            Further parameters are passed to xr.open_zarr
+        """
+        from intake_xarray.xzarr import ZarrSource
```

Comment:
Can we move this import to the beginning of the file?

Reply:
No! We don't want to import until necessary, because `import intake` would also import, and so take much longer.